pages/views/redirect/esl-a11y-group.njk

```diff
@@ -0,0 +1,5 @@
+---
+layout: forward
+target: https://esl-ui.com/components/esl-a11y-group/
```

Comment:
Should be relative.
We don't know if we going to change our domain + it should work locally with local domain